### DISCUSSION TITLE
When's a doc coming?

### DISCUSSION THREAD

**Discussion by Nasko-5** [Textualize/textual#237] [General]
I'm sort of confused and don't know where to look for help with textual, as no tutorials exist online, as far as I'm aware.
I've looked at the help in the README but i still cant figure out what the examples in the repo are doing
there are probably other people who are in a similar situation, i just haven't seen anyone make a discussion about it yet.

so, when is documentation for textual coming?

**Comment by jeffwright13** [comment]
I wonder if we might make this a community effort?

**Reply by tusharsadhwani** [reply]
I'd be more than willing to dive into the library and write a bunch of tutorials and maybe even a course. But only once there's a relatively stable API. cc @willmcgugan

**Reply by willmcgugan** [reply]
And that would no doubt be awesome! But there are some big changes coming, that will make the API far simpler.

Bear with us!

**Reply by tusharsadhwani** [reply]
Keep us updated here :)

**Reply by dissolved** [reply]
@willmcgugan is there an estimate of when these "big changes" are coming and or can you point me to the planned changes?

**Reply by willmcgugan** [reply]
You could check out the css branch.

**Comment by drrosa** [comment]
Hi @willmcgugan! I would recommend that you consider filling out [this application](https://goo.gle/3dRyD7P) so that Textualize can participate in Google's 2022 Season of Docs program (**deadline is March 25th**). If accepted, Textualize will receive grant money to hire someone that will focus on documentation efforts for 6 months. I participated as a technical writer in 2019 and had a good experience so I'm considering doing it again this year :)

**Reply by tusharsadhwani** [reply]
+1 on GSoD, it's a really good program

**Comment by tusharsadhwani** [comment]
Consider setting up textual.readthedocs.io whenever possible, if you use automation like [mkdocstrings](https://github.com/mkdocstrings/mkdocstrings) it should take no time at all.
